sift/client.py

```diff
@@ -502,6 +502,46 @@ def get_order_decisions(self, order_id, timeout=None):
             raise ApiException(str(e))
 
 
+    def apply_session_decision(self, user_id, session_id, properties, timeout=None):
+        """Apply decision to session
+
+        Args:
+            user_id: id of user
+            session_id: id of session
+            properties:
+                decision_id: decision to apply to user
```

Comment:
I guess `decision_id` here is "decision to apply to session" (and that the same comment in `apply_order_decision` should be changed accordingly.)

For example, in the unit test, `decision_id` used was `session_looks_bad_ato`.